integration-test/common/src/test/java/com/tencent/rss/test/ShuffleServerWithMemoryTest.java

```diff
@@ -153,8 +155,12 @@ public void memoryWriteReadTest() throws Exception {
 
     // send data to shuffle server, flush should happen
     List<ShuffleBlockInfo> blocks2 = createShuffleBlockList(
-        shuffleId, partitionId, 0, 3, 50,
-        Roaring64NavigableMap.bitmapOf(), dataMap, mockSSI);
+      shuffleId, partitionId, 0, 3, 50,
+      Roaring64NavigableMap.bitmapOf(), dataMap, mockSSI);
+    expectBlockIds.addLong(blocks2.get(0).getBlockId());
+    expectBlockIds.addLong(blocks2.get(1).getBlockId());
+    expectBlockIds.addLong(blocks2.get(2).getBlockId());
+    expectBlockIds.addLong(blocks.get(2).getBlockId());
```

Comment:
why update expectBlockIds? it is not used again

Reply:
The reference of expectBlockIds is delivered to ComposedClientReadHandler, and it is used fo the filter.

Reply:
I will move the update of expectBlockIds to block creation.
And update processBlockIds after each readShuffleData(), like the way in memoryAndLocalFileReadWithFilterTest.